QRCode/QRCode.swift

```diff
@@ -63,30 +63,32 @@ public struct QRCode {
     }
     
     public init?(_ url: NSURL) {
-        if let data = url.absoluteString?.dataUsingEncoding(NSISOLatin1StringEncoding) {
+        if let data = url.absoluteString.dataUsingEncoding(NSISOLatin1StringEncoding) {
             self.data = data
         } else {
             return nil
         }
     }
-
+    
     // MARK: Generate QRCode
     
     /// The QRCode's UIImage representation
-    public var image: UIImage {
-        return UIImage(CIImage: ciImage)!
+    public var image: UIImage? {
+        return UIImage(CIImage: ciImage!)
     }
```

Comment:
ciImage should not be forced unwrapped here, we should probably guard against that here as well.